src/api/Annotations.js

```diff
@@ -115,7 +133,7 @@ export default class Annotations extends MarkerBasedApi {
                 },
             },
             errorCallback,
-            successCallback,
+            successCallback: this.annotationSuccessCallback.bind(this, successCallback),
```

Comment:
Should we bind this once in the constructor?

Reply:
I'm not sure how would this work. I need to bind the `successCallback` (which is given as param in `updateAnnotation`) as the first param in order to invoke it in `annotationSuccessCallback`.